mona_sdk/client.py

```diff
@@ -667,8 +674,14 @@ def get_config_history(self, number_of_revisions=UNPROVIDED_VALUE):
         documentation here:
         https://docs.monalabs.io/docs/retrieve-config-history-via-rest-api
         """
-        return self._app_server_request(
-            "get_config_history", data={"number_of_revisions": number_of_revisions}
+        app_server_response = self._app_server_request("get_config_history")
```

Comment:
did you remove the number of revisions from the input data here deliberately?

Reply:
I added it back